### PR TITLE
feat: point in time connections in the "cs" command

### PR DESCRIPTION
**Pull request by duskpoet**
https://www.loom.com/share/91962335b5884c0cad4e02a9ed664a58

### CODE REVIEW

File: src/commands/connection_string.ts [hunk 1]
```diff
@@ -13,8 +14,17 @@ export const describe = 'Get connection string';
 export const builder = (argv: yargs.Argv) => {
   return argv
     .usage('$0 connection-string [branch] [options]')
+    .example('$0 cs main', 'Get connection string for the main branch')
+    .example(
+      '$0 cs main@2024-01-01T00:00:00Z',
+      'Get connection string for the main branch at a specific point in time',
```
Comment (danieltprice) [inline src/commands/connection_string.ts:20]:
```suggestion
      'Get a connection string for the primary branch at a specific point in time',
```

File: src/commands/connection_string.ts [hunk 2]
```diff
@@ -13,8 +14,17 @@ export const describe = 'Get connection string';
 export const builder = (argv: yargs.Argv) => {
   return argv
     .usage('$0 connection-string [branch] [options]')
+    .example('$0 cs main', 'Get connection string for the main branch')
```
Comment (danieltprice) [inline src/commands/connection_string.ts:17]:
```suggestion
    .example('$0 cs main', 'Get a connection string for the primary branch')
```

File: src/commands/connection_string.ts [hunk 3]
```diff
@@ -13,8 +14,17 @@ export const describe = 'Get connection string';
...
+    .example(
+      '$0 cs main@2024-01-01T00:00:00Z',
+      'Get connection string for the main branch at a specific point in time',
+    )
+    .example(
+      '$0 cs main@0/234235',
+      'Get connection string for the main branch at a specific LSN',
```
Comment (danieltprice) [inline src/commands/connection_string.ts:24]:
```suggestion
      'Get a connection string for the primary branch at a specific LSN',
```

File: src/commands/connection_string.ts [hunk 4]
```diff
@@ -13,8 +14,17 @@ export const describe = 'Get connection string';
...
+    )
     .positional('branch', {
-      describe: 'Branch name or id. If ommited will use the primary branch',
+      describe: `Branch name or id. If ommited will use the primary branch. Can be written in the point-in-time format: "branch@timestamp" or "branch@lsn"`,
```
Comment (danieltprice) [inline src/commands/connection_string.ts:27]:
```suggestion
      describe: `Branch name or id. Defaults to the primary branch if omitted. Can be written in the point-in-time format: "branch@timestamp" or "branch@lsn"`,
```

**Review by danieltprice — APPROVED** [review]
Just one minor suggestion. I initially thought mention of `main` should be changed to primary, but I see that's just an example branch name, which is valid.

**Review by Shridhad — APPROVED** [review]

**Pull request merged**